packages/driver/cypress/integration/commands/actions/click_spec.js

```diff
@@ -1224,6 +1231,42 @@ describe('src/cy/commands/actions/click', () => {
         })
       })
 
+      it('can specify scrollBehavior bottom in config', { scrollBehavior: 'bottom' }, () => {
```

Comment:
So I'm not super fond of having these three very similar tests in four different specs; They're not identical, but close enough that it feels redundant - like what we're really after is a test of scrollIntoView, rather than check/click/trigger/type.

Would it make sense to have a new spec / describe block with a single copy of these tests?

Reply:
I'd agree if they were unit tests. Since they're integration tests and, while there may be a good amount of overlap/redundancy, it's valuable to have coverage for each command, so I think it makes sense to keep them with each command's tests.